lib/hanami/commands/new/abstract.rb

```diff
@@ -93,6 +93,10 @@ def git_dir_present?
           File.directory?(target.join('.git'))
         end
 
+        def git_available?
+          system('which', 'git', out: File::NULL)
```

Comment:
Question: in order to have it work both on Unix and Windows, why can't we just use `system('git --version')`? It would return `true` if git is available and `nil` if it's not. If I recall correctly, we used it widely in the code that was supposed to work on both systems.